<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="QTextEdit" name="te_time"/>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="pb_circle">
      <property name="text">
       <string>Круг</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QPushButton" name="pb_start_stop">
      <property name="text">
       <string>Старт/стоп</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lb_time">
      <property name="text">
       <string>Время</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pb_clear">
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>te_time</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pb_clear</sender>
   <signal>clicked()</signal>
   <receiver>lb_time</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>18</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
